lib/help.js

```diff
@@ -67,4 +67,7 @@ function helpCommand(commands, commandArgs) {
   });
 }
 
-module.exports = helpCommand;
+module.exports = {
+  helpCommand,
+  findCommandByName,
+};
```

Comment:
I don't see a change in `paragon-scripts.js` to account for this. Wouldn't

https://github.com/openedx/paragon/blob/5162c89adc46bcf4d0faa1c0473ad17447915818/bin/paragon-scripts.js#L4

need to be updated?

Reply:
Good catch! Corrected